src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/cumulative-metrics-transition-guide.mdx

```diff
@@ -0,0 +1,151 @@
+
+* Take the same general approach we describe for alerts: create a new chart with an updated query and erase the old chart after the EoL, OR
+* Wait until after the EoL and replace the queries
+
+## Suggested action: Create new versions of alerts [#alerts]
+
+Here are some examples of queries that might be used in alerts. We'll give you the original query, then a `gauge` version of the query, and a `cumulativeCount` version of the query. You should replace your existing alert with the `gauge` version of the query, and create a duplicate alert condition that uses the `cumulativeCount` version. That way, when we transition your account to `cumulativeCount` on June 16, that alert condition will be available when the old query no longer works. 
+
+### Cumulative queries [#cumulative-queries]
+
+Original query: 
+```
+FROM Metric SELECT latest(<metricName>)
+```
+
+Gauge version of the query:
+``` 
+FROM Metric SELECT latest(<metricName>) 
+WHERE <metricName>[type] = 'gauge'
+```
+
+Cumulative count version of the query: 
+```
+FROM Metric SELECT latest(<metricName>[cumulative]) 
+WHERE <metricName>[type] = 'cumulativeCount'
+```
+
+### Rate of change queries [#rate-of-change-queries]
+
+Original query: 
+
+```
+FROM Metric SELECT derivative(<metricName>, 1 minute)
+```
+
+Gauge version of query:
+``` 
+FROM Metric SELECT derivative(<metricName>, 1 minute) 
+WHERE <metricName>[type] = 'gauge'
+```
+
+Cumulative count version of query:
+
+```
+FROM Metric SELECT rate(sum(<metricName>[cumulative]), 1 minute) 
+WHERE <metricName>[type] = 'cumulativeCount'
+```
+
+## Suggested action: preview cumulativeCount before the EoL [#preview]
+
+Affected accounts will continue to have their cumulative sums stored as `gauge` types until the EoL. However, optionally, you can try out `cumulativeCount` before the EoL to validate your queries and alerts are behaving as expected. To test `cumuativeCount` types early, add the following attribute to OTLP data you send to New Relic:
+
+```
+newrelic_metric_type=cumulativeCount
+```
+
+<Callout variant="important">
+When you start sending `newrelic_metric_type=cumulativeCount` on your data, any cumulative sums we receive will be translated into `cumulativeCount` types. This means any existing queries or alerts set up to target a `gauge` type will no longer be valid. Keep this in mind when testing the `cumulativeCount` experience. To return to the `gauge` type, simply stop setting the `newrelic_metric_type` attribute. 
+</Callout>
+
+### Instrumentation library [#instrumentation-library]
+
+If you're using an OTLP instrumentation library within your code base, you should reference the [appropriate language SDK](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-set-up-your-app/#instrument) for your application. Any method of adding the attribute `newrelic_metric_type=cumulativeCount` to individual metric data points or to the `Resource attribute` level should be sufficient. 
```

Comment:
```suggestion
If you're using an OTLP instrumentation library within your code base, you should reference the [appropriate language SDK](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-set-up-your-app/#instrument) for your application. Any method of adding the attribute `newrelic_metric_type=cumulativeCount` to individual metric data points or to the resource attribute level should be sufficient. 
```

I think referring to this in plain text makes more sense. In the draft I think I used monospace so that is my fault 😅